src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -1143,6 +1148,17 @@ private boolean undeclaredWriteInsEnabled() {
     return includeUwi;
   }
 
+  private boolean isOverVoteLabelEnabled() {
+    boolean overVote = false;
```

Comment:
super nit, but elsewhere in the code we refer to it using a single word ("overvote"), so these should be `isOvervoteLabelEnabled` and `overvote`